### PR TITLE
feat: redesign of environments

### PR DESCRIPTION
**Pull request by vfried**
Redesign of interactive environments

Normal:
image

Compact:
image

Inside project: (might be adjusted/reviewed in the future)
image

/deploy

### CODE REVIEW

**Review by ciyer — CHANGES_REQUESTED** [review]
I like how you have the design to this page. I have a few cosmetic suggestions.

## New Environment Page

There needs to be a gap between the "Start Environment" button and the footer.

Screen Shot 2021-03-31 at 1 54 59 PM

## Global Environments Page

I think the environments information should be shown in two columns, using **bold** for the titles (no colons). I also think the info and link to the commit should be merged into one element that is a popup. 

image

## Project Environments Page

Basically the same as the global version, just without the project name.

image

**Comment by vfried** [comment]
I did some changes, as discussed before, the form was not addressed in this PR.

This is how it looks
image

image

In my opinion, this is much cleaner now, what do you think?

**Review by ciyer — APPROVED** [review]
I agree, it looks fantastic now!
